### PR TITLE
Fix MSP_BOARD_INFO accumulation and localStorage Quota Exceeded

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: #2938 

- [x] Fixes `FC.CONFIG.boardName` parameter accumulating previous values.
- [x] Move `Loader` function in a `try catch error` statement.
- [x] Move `Jenkins` + firmware `cache` from `localStorage` to new `SessionStorage` to fix `Exceeded Quota` error.
- [x] Remove old keys from LocalStorage.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
Looks like we also have a quota issue.
```
    quota: function() {
        let data = '';
        for(const key in window.localStorage){
            if (window.localStorage.hasOwnProperty(key)) {
                data += window.localStorage[key];
            }
        }

        return data ? 3 + ((data.length * 16) / (8 * 1024)) : 0;
    },
```

File: src/js/SessionStorage.js
Comment (limonspb) [inline src/js/SessionStorage.js:12]: console.error maybe?

File: src/js/SessionStorage.js
Comment (limonspb) [inline src/js/SessionStorage.js:21]: console.error?

File: src/js/SessionStorage.js
Comment (limonspb) [inline src/js/SessionStorage.js:35]: console.error

**Review by limonspb — CHANGES_REQUESTED** [review]

File: src/js/SessionStorage.js
Comment (haslinghuis) [reply src/js/SessionStorage.js]: Fixed

File: src/js/SessionStorage.js
Comment (haslinghuis) [reply src/js/SessionStorage.js]: Fixed

File: src/js/SessionStorage.js
Comment (haslinghuis) [reply src/js/SessionStorage.js]: Fixed

**Review by limonspb — APPROVED** [review]

**Comment by McGiverGim** [comment]
What happens with old values stored in the local storage? They will remain there until the user deletes the folder in the computer?

**Review by chmelevskij — DISMISSED** [review]

**Comment by haslinghuis** [comment]
Old values will remain until user removes the folder.

**Review by limonspb — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]
The cleanup is very simple, maybe we need to store some "version" in the localStorage, and act according to the change of version. But for the moment, I think is enough with your code. I suppose this is a exceptional situation and we can remove the cleaning in one or two versions.

**Review by Asizon — APPROVED** [review]

**Pull request merged**
